app/views/client/home/_ukm.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="ukm">
+	<div class="ukm__title-wrap">
```

Comment:
start class container-medium from here

Reply:
remove title wrap, and add styling to ukm__title

Reply:
okay, already to adding container-medium, and deleted wrap-title class